app/src/main/java/org/jellyfin/mobile/bridge/NativeInterface.kt

```diff
@@ -131,7 +132,9 @@ class NativeInterface(private val activity: MainActivity) {
             Timber.e("Download failed: %s", e.message)
             return false
         }
-        activity.requestDownload(Uri.parse(url), title, filename)
+        runBlocking {
```

Comment:
What thread does the code run on? Can `runBlocking` possibly cause ANRs? Not sure myself, that's why I'm asking.

Reply:
I'm not sure what thread it uses but the idea is that it will make the JavaScript function blocking (like `alert`) until the coroutine finishes. In my testing it worked fine.

Reply:
Ok, I'll merge this then.